checker-qual/src/main/java/org/checkerframework/framework/qual/UpperBoundFor.java

```diff
@@ -18,7 +18,7 @@
  * An example is the declaration
  *
  * <pre><code>
- * {@literal @}DefaultFor(classes=String.class)
+ * {@literal @}UpperBoundFor(classes=String.class)
```

Comment:
Instead of `classes` this must be either `typeKinds` or `types` here - given the `String.class` it should be `types`.
Can you check whether that is wrong in `DefaultFor`'s javadoc?